scripts/start_kafka.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+source $(dirname $0)/kafka_base_dir.sh
+BUILD_DIR=$(pwd)/_build/kafka
+
+mkdir -p $BUILD_DIR
+
+for i in 1 2 3
+do
+  port=$(expr 9092 + ${i} - 1)
```

Comment:
SRP in Bash. Both loops do one thing ;-). I thought about it, but this seemed cleaner in terms of flow through the script.